test/conformance/channel_tracing_test.go

```diff
@@ -0,0 +1,163 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conformance
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"knative.dev/eventing/test/base/resources"
+	"knative.dev/eventing/test/common"
+	"knative.dev/eventing/test/conformance/helpers"
+	"knative.dev/pkg/test/zipkin"
+)
+
+func TestChannelTracing(t *testing.T) {
+	testCases := map[string]struct {
+		incomingTraceId bool
+		istio           bool
+	}{
+		//"no incoming trace id": {},
```

Comment:
Ah, so this one is still not working? :)

Reply:
Works consistently on my Linux desktop, is flaky on my Mac laptop. For now, I'll leave it commented out (or I could remove the line if you prefer), so that the rest of the test can get in.

Reply:
Removed it for now, rather than just commenting it out.